viewer/src/components/Commands.vue

```diff
@@ -654,6 +673,13 @@ export default class Commands extends Vue {
 
   destroyed() {
     window.removeEventListener("keydown", this.keyListener);
+    this.unsubscribe();
+  }
+
+  private unsubscribe() {
+    for (const s of Object.values(this.subscriptions)) {
+      s();
+    }
```

Comment:
I don't think it will cause any issues - but to keep memory clean, add this after:

```
this.subscriptions = {}
```

That way `this.subscriptions` will always reflect the active subscriptions, if we need that info for any purpose